### ISSUE TITLE
[Other] Video in About Us section 

### ISSUE THREAD

**Issue by Vin205** [Rakesh9100/Beautiify#1213]
### What would you like to share?

A video should be included in about us section about the working of website so that users can easily understand the purpose of the website and make use of it efficiently.

### Additional information

It is easier for users to easily understand the working of website.
![Screenshot 2024-07-11 173346](https://github.com/Rakesh9100/Beautiify/assets/138381350/3ae0767b-b479-4267-8005-0ed12b68ba22)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by saurabhsingh720** [comment]
@Vin205 
What would u want to show in video?

**Comment by Vin205** [comment]
Video about beautify and how to use components for styling website, how beautify can help you to beautify your webiste and make website responsive, How UI can be enhanced with beautify.

**Comment by Rakesh9100** [comment]
So, do you want to just do the screen recording of the website and attach it to the About page @Vin205

**Comment by Vin205** [comment]
No, not just screen recording, I will also show how I will we using beautify components to develop new website. I will create a website with beautify components and that video I will include in About us page

**Comment by Vin205** [comment]
please assign me this issue
